packages/components/InputPanel/NumericButton.tsx

```diff
@@ -0,0 +1,18 @@
+import { GridItem } from '@chakra-ui/react';
+
+import BaseButton from '../BaseButton';
+
+interface Props {
+  children: string;
+  onclick: () => void;
+}
+
+export const NumericButton = (props: Props) => {
+  const { children, onclick } = props;
+  const colStart = children === '0' ? 2 : undefined;
```

Comment:
n entendi mto bem isso n

Reply:
what part specifically?

Reply:
tbh i don't understand how is this prop is acting here , if you could explain it would be nice!

Reply:
The colStart property indicates what column in the grid a grid item should start to occupy, there is also the property colEnd, which indicates which column the grid item should be ended. Here I'm indicating that, if the `children` (the button text) is equal to `0` this grid item should start and ended (implicitly) in grid column `2`, elsewhere we didn't specify (`undefined`) and so the grid applies its default occupation rule.

Please, let me know if I answered your question, or if your point was about a different point :)

Reply:
got it